core/features/player/commands/create/create_player_cmd_serializer.py

```diff
@@ -29,8 +28,14 @@ class CreatePlayerCommandSerializer(serializers.Serializer):
             lambda value: validate_format(value)
         ]
     )
+    
+    password = serializers.CharField(
+        validators=[lambda value: validate_non_empty(value), 
+                    lambda value: validate_min_length(value, 8), 
```

Comment:
remove validate_min_length and validate_max_length.

- Both are checked by "validate_password_format".